Added example implementation.

mrchantey opened the pull request.

simonihmig commented on README.md:
```diff
@@ -2,4 +2,162 @@
+ECS design will help us to write more organized apps by introducing some strict rules as to how they shall be structured:
+1. Conceptual elements of the app are organized into **entities**
+2. All game state and data are fields of **components**
+3. All behaviour exits in **systems**
```
```suggestion
3. All behaviour exists in **systems**
```

simonihmig commented on README.md:
````diff
@@ -2,4 +2,162 @@
+In keeping with rule 1, lets use ecsy-babylon to convert our existing objects to components on entities.
+
+```ts
+import * as EcsyBabylon from "ecsy-babylon";
````
I would *slightly* prefer importing only the needed parts here, but that's not a deal-breaker for sure...

simonihmig reviewed (commented):
Thanks a lot for your effort here, that's awesome! :+1:

Apart from the inline comments, could you please tweak the code examples just a little bit for consistent style:
* semicolons at end of line everywhere
* consistent indentation of 2 spaces

In the repo there exists already two simple [demo app](https://github.com/kaliber5/ecsy-babylon/tree/master/demos), would be nice to add your demo app there also, so we can make sure the demo app is actually working (e.g. when future breaking changes are introduced), and to make the readme link to the source code in this repo. The existing demo apps are built with rollup instead of webpack, but copying the existing setup should be easy.

So bonus points for adding that, but that can also be done in a separate PR, or if you don't have the time for that I can do that also eventually later...

mrchantey commented:
Ok no worries, I've added semicolons and fixed 'exits'. Couldn't figure out how to build or watch from inside the demo folder so just copied the source over there and added the rollup.config from the boxes demo. The 'source' link at the bottom of the readme now points to where I assume the source will be after the PR goes through. Wasn't sure about individual imports because the name overwrites may confuse people at first glance, ie `ecsyBabylon.World` vs `ecsy.World` or `ecsyBabylon.ArcRotateCamera` vs `BABYLON.ArcRotateCamera`, I'll leave that one with you.

simonihmig commented:
@mrchantey Thanks again for working on this, and sorry for the long delay here!

I'll merge this, will fix the broken linting in your demo app later...
